.idea/vcs.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/ingsw-group1-message-library" vcs="Git" />
```

Comment:
Questa cartella mi turba relativamente. Non vorrei che qualcosa andasse a quel paese con i submodule.

Reply:
Credo significhi che hai ancora la repo della libreria sms fuori dalla repo della rete replicata, non so onestamente quale opzione sia meglio, se tenere i submodule dentro o fuori altri submodule, chiederò al Professore o al Dr. Li

Reply:
Sì mi è rimasta fuori, ma è segnata in rosso perchè fuori dalla staging area e sono stato attento a tenerla così, strano che sia comparsa però🤨